config/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+full_path=$(realpath $0)
+dir_path=$(dirname $full_path)
+
+jar_file="$dir_path/google-java-format-1.24.0-all-deps.jar"
```

Comment:
could you reference the jar file downloaded by maven instead? then we wouldn't have to commit the jar file, something like
~/.m2/repository/com/google/googlejavaformat/google-java-format/1.24.0/google-java-format-1.24.0.jar

Reply:
We can't use the script from .m2 because it needs the -all-deps.jar